plugins/CorePluginsAdmin/vue/src/FormField/FieldSite.vue

```diff
@@ -17,6 +17,7 @@
         :switch-site-on-select="false"
         :show-selected-site="true"
         :only-sites-with-admin-access="uiControlAttributes.onlySitesWithAdminAccess || false"
+        :sites-with-at-least-write-access="uiControlAttributes.sitesWithAtLeastWriteAccess || false"
```

Comment:
The option is named this way:
```suggestion
        :only-sites-with-at-least-write-access="uiControlAttributes.onlySitesWithAtLeastWriteAccess
         || false"
```

Reply:
Thank you for catching this. :1st_place_medal:  I thought that I had covered all the renaming.